static/js/faculty_carousel.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Slider from 'react-slick';
+
+const carouselDiv = document.querySelector('#faculty-carousel');
+
+class FacultyTile extends React.Component {
+  render() {
+    let nameStr;
+    if (this.props.title) {
+      nameStr = this.props.name + ", " + this.props.title
+    } else {
+      nameStr = this.props.name
+    }
+    let style = {'backgroundImage': 'url('+this.props.imageUrl+')'};
```

Comment:
I _believe_ that React will take care of the escaping here, but I'm not sure. Can you give me an example of a string that will cause an error, if not properly handled/escaped?